### ISSUE TITLE
Statistics graphs not updating

### ISSUE THREAD

**Issue by madprime** [openSNP/snpr#515]
https://opensnp.org/statistics the graphs seem to be ending a year ago?

Or maybe they're actually up-to-date and the X axis labels are off.

**Comment by gedankenstuecke** [comment]
Yeah, this is a bug on our deployment where the cronjobs for running the updates don't run right now. :(
